apps/style/netsim/style.scss

```diff
@@ -410,12 +415,16 @@ svg {
       }
 
       .watermark {
-        color: #d3d3d3;
+        color: $medium_gray;
       }
 
-      .bit_counter {
+      .bit-counter {
         text-align: right;
-        color: darkgray;
+        color: $dark_gray;
+      }
+
+      .bit-counter.oversized {
```

Comment:
Merge rule `.bit-counter.oversized` with rule on line 421<br>Selector should have depth of applicability no greater than 2, but was 4